light_curves/light_curve_classifier.md

```diff
@@ -95,7 +94,6 @@ from sklearn.metrics import ConfusionMatrixDisplay, accuracy_score, confusion_ma
 from sklearn.model_selection import train_test_split
 from sklearn.gaussian_process import GaussianProcessRegressor
 from sklearn.gaussian_process.kernels import RBF
```

Comment:
These imports and the ones in the next suggestion don't appear anywhere else except in this cell. (I didn't touch the ones that do appear later but are commented out.) While developing, it might make sense to leave them in if you think you might want them. If you're basically done developing, it's probably time to either remove them or add a commented-out line of code showing how they would be used.